src/functional/iterator/pretty_printer.py

```diff
@@ -4,6 +4,8 @@
 """
 from __future__ import annotations
 
+import inspect
+import textwrap
```

Comment:
Since you only use one function from this package once, do you want to import only that rather than the entire package?